website/_dogfooding/_pages tests/partials-tests.mdx

```diff
@@ -0,0 +1,12 @@
+import PagePartial from './_pagePartial.mdx';
```

Comment:
our dogfood section is messy currently.

I'd prefer if you created this in `docusaurus/website/_dogfooding/_docs tests/toc-partials` and added much more complex cases such as "deeply-nested partials" (one partial importing another)